patch/config-quanta-ix1b.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+index 3d31fb2..97827d4 100644
+--- a/debian/build/build_amd64_none_amd64/.config
++++ b/debian/build/build_amd64_none_amd64/.config
+@@ -5128,6 +5128,7 @@ CONFIG_LEDS_CLASS=y
+ CONFIG_LEDS_PCA9532=m
+ # CONFIG_LEDS_PCA9532_GPIO is not set
+ # CONFIG_LEDS_GPIO is not set
```

Comment:
remove CONFIG_LEDS_GPIO is not set

scripts/kconfig/conf --silentoldconfig Kconfig
.config:5132:warning: override: reassigning to symbol LEDS_GPIO

Reply:
already removed the line, please checked, thanks.